plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenter.java

```diff
@@ -29,63 +33,104 @@
 public class SubversionProjectImporterPresenter extends AbstractWizardPage<ProjectConfigDto>
         implements SubversionProjectImporterView.ActionDelegate {
 
-    private SubversionProjectImporterView view;
+    // start with white space
+    private static final RegExp SUBVERSION_REPOSITORY_REGEX =
+            RegExp.compile("^(http|https|svn|svn\\+ssh)://[A-Za-z0-9_\\-]+(\\.[A-Za-z0-9_\\-:]+)+(/[A-Za-z0-9_.\\-]+)*/?$");
```

Comment:
It's tested with both wrong and correct values in [SubversionProjectImporterTest](https://github.com/stour/che/blob/CHE-864/plugins/plugin-svn/che-plugin-svn-ext-ide/src/test/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenterTest.java#L127-L163) yes.